backend/src/controllers/utils/index.ts

```diff
@@ -72,3 +81,186 @@ export const getTemplatesPath = (): string => {
   }
   return './templates/';
 };
+
+
+export const createXml = (savedInvoice: IInvoice, companyConfig: ICompanyConfig): string => {
+  const invoiceXml = new Invoice(savedInvoice.number.toString(), {
+    //This empty object is created to keep TypeScript from complaining, it has no influence on the generated xml
+    issuer: {
+      prefix: '',
+      endDate: '',
+      endRange: '',
+      resolutionNumber: '',
+      startDate: '',
+      startRange: '',
+      technicalKey: ''
+    },
+    software: {
+      id: '',
+      pin: '',
+      providerNit: ''
+    }
+  });
+
+  if (savedInvoice && companyConfig) {
+    const currencyID = { currencyID: DEFAULT_CURRENCY };
+    const taxSchemeIDVAT = new TaxScheme({ id: 'VAT' });
+    const customerCountryAndCode = COUNTRY_CODES.find(codes => codes.country === savedInvoice.client.country || codes.code === savedInvoice.client.country);
+
+    let cityRef = savedInvoice.client.city.trim();
+    const customerPostalAddress = new PostalAddress({
+      streetName: savedInvoice.client.address,
+      cityName: cityRef,
+      country: new Country({ identificationCode: customerCountryAndCode ? customerCountryAndCode.code : DEFAULT_COUNTRY_CODE })
+    });
+
+    cityRef = companyConfig.company.city.trim();
+    const supplierPostalAddress = new PostalAddress({
+      streetName: companyConfig.company.address,
+      cityName: cityRef,
+      country: new Country({ identificationCode: DEFAULT_COUNTRY_CODE })
+    });
+    const supplierLegalEntity = new PartyLegalEntity({
+      registrationName: companyConfig.company.name,
+      companyID: companyConfig.company.btw
```

Comment:
Are we sure that the CompanyID is the VAT nr?

Reply:
CBE == BCE == KBO (en in company model == company.btw)

CBE is the english translation of KBO